support-frontend/assets/pages/[countryGroupId]/checkout/components/PersonalDetailsFields.tsx

```diff
@@ -45,6 +55,9 @@ export function PersonalDetailsFields({
 					}}
 					onBlur={(event) => {
 						event.target.checkValidity();
+						if (confirmedEmail.length > 0) {
+							confirmEmailRef.current?.querySelector('input')?.checkValidity();
```

Comment:
Just a question with this logic, we added it to guard against a user dong the following:

- Enter a value email `george@theguardian.com`
- Enter a value in confirm email `george@theguardian.com` 
- Change value in to email `andrew@theguardian.com`

With this is ref the validation on `confirmEmailRef` will alert there's a mismatch immediately after the user enters `andrew@theguardian.com`. 

If we hadn't implemented this `ref` logic would this mismatch have been picked up on form submit and prevented submission?

Reply:
Just tested - yes that's correct. This is just doing the validation sooner on the email fields onBlur

Reply:
yeah it's a fairly edgecase situation, that simplification would be better 👍